Cake.Recipe/Content/build.cake

```diff
@@ -178,10 +179,26 @@ BuildParameters.Tasks.BuildTask = Task("Build")
 BuildParameters.Tasks.DotNetCoreBuildTask = Task("DotNetCore-Build")
     .IsDependentOn("Show-Info")
     .IsDependentOn("Print-AppVeyor-Environment-Variables")
-    .IsDependentOn("DotNetCore-Clean")
+    .IsDependentOn("Clean")
     .IsDependentOn("DotNetCore-Restore")
     .Does(() => {
-        Information("DotNetCore-Build...");
+        Information("Building {0}", BuildParameters.SolutionFilePath);
+
+        DotNetCoreBuild(BuildParameters.SolutionFilePath.FullPath, new DotNetCoreBuildSettings
+        {
+            Configuration = BuildParameters.Configuration,
+            ArgumentCustomization = args => args
+                .Append("/p:Version={0}", BuildParameters.Version.SemVersion)
+                .Append("/p:AssemblyVersion={0}", BuildParameters.Version.Version)
+                .Append("/p:FileVersion={0}", BuildParameters.Version.Version)
```

Comment:
@philo Yes agreed. I was going to do that. I borrowed this code from the Cake PR, and hadn't yet looked whether this variable could be passed in the same way.